### PR TITLE
Fixed bad typing in event delegate

### PR DESCRIPTION
**Pull request by brifsttar**
Hi Chad,

This fix is relative to #3. I had never tested Inlets before, and found on issue while making an example BP for the issue (for some reason, arrays needs to be passed by reference in event delegates).

Bertrand

### CODE REVIEW

**Review by cboulay — APPROVED** [review]

**Comment by cboulay** [comment]
Thanks. And I've given you write access to the repo so you can work directly here instead of on your fork if you like. If you make a big commit then it's still a good idea to make a PR so someone else (me) can get eyes on it. But realistically I won't be installing UE4 anytime soon so I will trust you to make sure things work before you submit a PR (or make it clear that the PR isn't working and shouldn't be merged _yet_).

**Comment by brifsttar** [comment]
Chad,

No problem, small bugfixes will get merged without PR, anything else will get a PR and approval from you.

**Pull request merged**
